covid19_npis/model/number_of_tests.py

```diff
@@ -117,8 +193,18 @@ def generate_testing(name_total, name_positive, modelParams, new_E_t):
         phi_plus=phi_t,
         phi_age=phi_age,
     )
+
+    positive_tests = yield weekly_modulation(
+        name="weekly_modulation",
+        modelParams=modelParams,
+        cases=positive_tests,
+        week_modulation_type="abs_sine",
+    )
+
     positive_tests = yield Deterministic(
-        name_positive, positive_tests, shape_label=("time", "country", "age_group")
+        name=name_positive,
+        value=positive_tests,
```

Comment:
Move  yield Deterministic into weekly_modulation to unclutter main loops

Reply:
done - though ... why does it need a "yield" then... this thing is really crunching my gears...